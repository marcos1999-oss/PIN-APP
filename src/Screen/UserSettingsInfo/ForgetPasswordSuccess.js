import React from "react";
import { Text, View, SafeAreaView, Dimensions } from "react-native";
import { Formik } from "formik";
import CustomIcon from "../../CustomIcon";
import { Btn, TextInput } from "../../Components/Common";
import Image from "react-native-auto-height-image";
import { city2 } from "../../Assets";
import { FormContainer } from "./Common";

/*Style*/
import style from "./style.js";
const countryCode = "800";
const networkProvider = "923";

export default class ForgetPasswordSuccess extends React.Component {
    constructor(props) {
        super(props);
    }

    form = {
        phone: `${countryCode} - ${networkProvider} - `,
    };

    submit = (values, actions) => {
        // alert(actions);
    };

    continuePress = () => {

    };

    render() {
        const deviceWidth = Dimensions.get("window").width;
        return (
            <SafeAreaView style={style.l_container}>
                <View style={{ marginTop: 30, alignItems: "center", marginBottom: 15 }}>
                    <Text style={{ fontSize: 14, color: 'white', marginBottom: 16 }}>
                        Success
                    </Text>
                    <CustomIcon name="setting" size={40} color={"#fff"} />
                </View>


                <View style={{ marginLeft: 25, marginTop: 16, marginBottom: 15 }}>
                    <Text style={{ fontSize: 16, color: 'white', marginBottom: 16 }}>
                        Support Team
                    </Text>
                    <Text style={{ fontWeight: 'bold', fontSize: 10, color: 'white', marginBottom: 16 }}>
                        You will receive a password reset email soon...
                    </Text>
                </View>


                <Image style={style.l_bottom} source={city2} width={deviceWidth} />

                <View style={{ marginLeft: 25, marginRight: 25, marginTop: 50 }}>
                    <Btn
                        color={'#FFFFFF'}
                        textColor={"#5F92F3"}
                        block
                        rounded
                        title="Continue"
                        onPress={this.continuePress}
                    />

                    <Text style={{ marginTop: 8, fontSize: 10, color: 'white', textAlign: 'center', marginBottom: 16 }}>
                        Please check you email
                    </Text>
                </View>
            </SafeAreaView>
        );
    }
}
