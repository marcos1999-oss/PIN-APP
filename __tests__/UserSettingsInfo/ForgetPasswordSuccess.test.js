/**
 * @format
 * @lint-ignore-every XPLATJSCOPYRIGHT1
 */

import 'react-native';
import React from 'react';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

import ForgetPasswordSuccess from '../../src/Screen/UserSettingsInfo/ForgetPasswordSuccess';

jest.mock('react-native-cached-images', () => {
  return {
    DocumentDir: () => {},
    ImageCache: {
      get: {
        clear: () => {},
      },
    },
  }
});

test('renders correctly', () => {
  const tree = renderer.create(<ForgetPasswordSuccess />).toJSON();
  expect(tree).toMatchSnapshot();
});
