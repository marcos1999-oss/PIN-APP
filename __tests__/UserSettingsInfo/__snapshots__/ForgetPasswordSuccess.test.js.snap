// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#5F92F3",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginBottom": 15,
        "marginTop": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 14,
          "marginBottom": 16,
        }
      }
    >
      Success
    </Text>
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#fff",
            "fontSize": 40,
          },
          undefined,
          Object {
            "fontFamily": "PinAppIcon",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      î¤’
    </Text>
  </View>
  <View
    style={
      Object {
        "marginBottom": 15,
        "marginLeft": 25,
        "marginTop": 16,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 16,
          "marginBottom": 16,
        }
      }
    >
      Support Team
    </Text>
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 10,
          "fontWeight": "bold",
          "marginBottom": 16,
        }
      }
    >
      You will receive a password reset email soon...
    </Text>
  </View>
  <Image
    onError={[Function]}
    onHeightChange={[Function]}
    source={
      Object {
        "testUri": "../../../assets/img/city2.png",
      }
    }
    style={
      Array [
        Object {
          "height": 0,
          "width": 750,
        },
        Object {
          "bottom": 0,
          "position": "absolute",
        },
      ]
    }
  />
  <View
    style={
      Object {
        "marginLeft": 25,
        "marginRight": 25,
        "marginTop": 50,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "stretch",
          "backgroundColor": "#FFFFFF",
          "borderBottomWidth": null,
          "borderColor": "#007aff",
          "borderLeftWidth": null,
          "borderRadius": 57,
          "borderRightWidth": null,
          "borderTopWidth": null,
          "borderWidth": undefined,
          "elevation": 2,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 6,
          "paddingTop": 6,
          "shadowColor": undefined,
          "shadowOffset": undefined,
          "shadowOpacity": undefined,
          "shadowRadius": undefined,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 24,
            },
            undefined,
            Object {
              "color": "#5F92F3",
            },
          ]
        }
      >
        Continue
      </Text>
    </View>
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 10,
          "marginBottom": 16,
          "marginTop": 8,
          "textAlign": "center",
        }
      }
    >
      Please check you email
    </Text>
  </View>
</RCTSafeAreaView>
`;
